java/server/test/org/openqa/selenium/grid/node/NodeTest.java

```diff
@@ -425,6 +425,17 @@ private File getTemporaryFilesystemBaseDir(TemporaryFilesystem tempFS) {
     return baseDir;
   }
 
+  //Test that the draining command sets Host status to DRAINING
+  @Test
+  public void drainingNodeStatusDraining() {
+
+  }
+
+  //Test that a draining node doesn't accept new sessions by any means
```

Comment:
?

Reply:
Oops, sorry my bad. I didn't realize this was there from the previous commits. I have fixed it. Thank you so much for the review.